checkers/src/Board.cpp

```diff
@@ -1,3 +1,12 @@
+/**
+ * Projekt Zaawansowane Programowanie w C++ - Warcaby
+ * 24.04.2020
+ * 
+ * Autorzy: Patrycja Cieplicka, Adam Napieralski
+ * 
+ * Plik źródłowy klasy Board, która reprezentuje plansze do gry w Warcaby
+ * 
+ * */
```

Comment:
Może jak cały kod mamy po angielsku to trzymajmy się też tego w komentarzach, a jedynie GUI po polsku, ale to taka drobnostka